app/views/pages/new_project.html.erb

```diff
@@ -0,0 +1,172 @@
+<!--
```

Comment:
Can we remove this comment?

We can add the tailwind forms plugin, either as part of this PR or in a separate PR, if necessary.

Reply:
of course we can remove the comment, it came with the example on the tailwind website. We don't need to add the forms plugin, it's already added actually:
https://github.com/opensourcecatholic/marriage-booklet/blob/bac858c23c48ec2b648ef830d4e7c1606a1abb0e/tailwind.config.js#L962-L966

Reply:
done in https://github.com/JohnRDOrazio/marriage-booklet/commit/15adb8e2a68257c50f8371de2a41ed5faea42bde